docs/reference/configuration/raftstore-config.md

```diff
@@ -36,24 +36,24 @@ Determines the maximum number of peers that a single thread can obtain in a batc
 
 **`raftstore.store_pool_size`**
 
-Determines the number of threads to process peers in batch. The value must be an positive integer. For better performance, we recommend you set a value less than or equal to the number of your CPU cores.
+Determines the number of threads to process peers in batch. The value must be a positive integer. For better performance, it is recommended that you set a value less than or equal to the number of CPU cores on your machine.
  
-**`apply_max_batch_size`**
+**`raftstore.apply_max_batch_size`**
 
-Determines the maximum number of peers that a single thread can resolve in a batch. The value must be a positive integer. A smaller value provides better load balancing for CPU, but may cause more frequent writes.
+Determines the maximum number of ApplyDelegates requests that a single thread can resolve in a batch. The value must be a positive integer. A smaller value provides better load balancing for CPU, but may cause more frequent writes.
 
-**`apply_pool_size`**
+**`raftstore.apply_pool_size`**
 
-Determines the number of threads. The value must be a positive integer. For better performance, we recommend you set a value less than or equal to the number of your CPU cores.
+Determines the number of threads. The value must be a positive integer. For better performance, it is recommended that you set a value less than or equal to the number of CPU cores on your machine.
 
 ## Hibernate Region
 
-Hibernate Region is a Raftstore feature to reduced the extra overhead caused by heartbeat messages between the Raft leader and the followers for idle regions. With this feature enabled, a region idle for a long time is automatically set as hibernated. The heartbeat interval for the leader to maintain its lease becomes much longer, and followers do not initiate a re-selection simply because they could not receive heartbeats from the leader.
+Hibernate Region is a Raftstore feature to reduce the extra overhead caused by heartbeat messages between the Raft leader and the followers for idle regions. With this feature enabled, a region idle for a long time is automatically set as hibernated. The heartbeat interval for the leader to maintain its lease becomes much longer, and followers do not initiate a re-selection simply because they cannot receive heartbeats from the leader.
```

Comment:
s/a re-selection/elections/

Reply:
Updated. PTAL and approve if it's good. Thanks.